lld/test/ELF/aarch64-reloc-gotpcrel32.s

```diff
@@ -0,0 +1,35 @@
+// REQUIRES: aarch64
+// RUN: llvm-mc -filetype=obj -triple=aarch64 %s -o %t.o
+// RUN: ld.lld %t.o -o %t.so -shared
+// RUN: llvm-readobj -S %t.so | FileCheck --check-prefix=SEC %s
+// RUN: llvm-objdump --no-print-imm-hex -s -d %t.so | FileCheck %s
+
+
+// SEC:         Name: .got (58)
```

Comment:
Use `llvm-readelf -S` to get section addresses in new tests. The tabular output is easier to read.

Reply:
Done